src/newLaunch/stages/stage8.html

```diff
@@ -1,63 +1,91 @@
-          <a class='wrappedLink' href="${ipfsURL}" target="_blank" rel="noopener noreferrer">${ipfsURL}</a>.
+        <p>
+          You may inspect the ${wizardState.launchTypeTitle} configuration, stored in the JSON format in IPFS, here:
+          <a class="wrappedLink" href="${ipfsURL}" target="_blank" rel="noopener noreferrer">${ipfsURL}</a>.
           <span class="importantText">Please save this URL.</span>
         </p>
 
-        <p>PrimeDAO Governance needs to approve and initialize your new ${wizardState.launchTypeTitle} before it can
-          appear in and be used in the Prime Launch interface. The approval takes place in the <a
-            href.bind="multiSigWalletUri" target="_blank" rel="noopener noreferrer">Prime Launch multisig wallet</a>.
-          Your ${wizardState.launchTypeTitle} may take up to 5 minutes to appear in the multisig. Reviews generally take
-          5-7 days."</p>
+        <p if.to-view="dataToken">
+          <span class="importantText"
+            >Note that when you generate a new token, the value will only be displayed once, so be sure to write it
+            down</span
+          >
+          <span class="dataTokenWrapper">${dataToken}</span>
+          <span>
+            <copy-to-clipboard-button
+              if.to-view="!hideClipboardButton"
+              text-to-copy.to-view="dataToken"
+              message.to-view="copyMessage"
+            ></copy-to-clipboard-button>
+            Copy to clipboard
+          </span>
+        </p>
+
+        <p>
+          PrimeDAO Governance needs to approve and initialize your new ${wizardState.launchTypeTitle} before it can
+          appear in and be used in the Prime Launch interface. The approval takes place in the
+          <a href.bind="multiSigWalletUri" target="_blank" rel="noopener noreferrer">Prime Launch multisig wallet</a>.
+          Your ${wizardState.launchTypeTitle} may take up to 5 minutes to appear in the multisig. Reviews generally
+          take 5-7 days."
+        </p>
 
-        <p>After being approved, the <etherscanlink hide-clipboard-button
-            address.to-view="wizardState.launchAdminAddress" text="${wizardState.launchTypeTitle} Admin">
+        <p>
+          After being approved, the
+          <etherscanlink
+            hide-clipboard-button
+            address.to-view="wizardState.launchAdminAddress"
+            text="${wizardState.launchTypeTitle} Admin"
+          >
           </etherscanlink>
 
           will have to fund the ${wizardState.launchTypeTitle} with
 
           <span if.bind="wizardState.requiredProjectTokenDeposit">
             <formatted-number
               value.to-view="wizardState.requiredProjectTokenDeposit  | ethwei:launchConfig.tokenDetails.projectTokenInfo.decimals"
-              thousands-separated></formatted-number>
+              thousands-separated
+            ></formatted-number>
             ${launchConfig.tokenDetails.projectTokenInfo.symbol}
```

Comment:
The symbol is not shown. Can you check that?
![image](https://user-images.githubusercontent.com/2517870/188846886-315fe05c-57e9-4289-bc95-61fd4937642c.png)

Reply:
hm so with D2D it's correct. i'll check rinkeby with GILAD
![image](https://user-images.githubusercontent.com/85691163/188858369-860d66fd-f79c-4615-b0cb-913741a01dfb.png)